src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -323,6 +323,11 @@ protected override INamespaceSymbol CommonCreateErrorNamespaceSymbol(INamespaceS
                        name).GetPublicSymbol();
         }
 
+        protected override IPreprocessingSymbol CommonCreatePreprocessingSymbol(string name)
+        {
+            return new Symbols.PublicModel.PreprocessingSymbol(name, CommonAssembly, CommonSourceModule);
```

Comment:
i never got a clear answer why we need assembly/module info in these symbols.

Reply:
A non-null assembly and module is required to be implemented by the symbol in FAR so that the scope of the FAR is determined. Cannot remember if that was customizable without adjusting the core implementation of FAR, but without the assembly and module, FAR completely fails to identify the project(s) to search in.

Reply:
> A non-null assembly and module is required to be implemented by the symbol in FAR so that the scope of the FAR is determined

This can be tweaked in FAR.